crates/swc/tests/tsc-references/expressions/optionalChaining/callChain/callChain/input.ts/es2015.1.normal/output.js

```diff
@@ -60,7 +60,7 @@ o2 === null || o2 === void 0 ? void 0 : o2["b"](1, ...[
     3
 ], 4).c;
 const v = o4 === null || o4 === void 0 ? void 0 : o4(incr);
-(ref12 = _obj = o5()) === null || ref12 === void 0 ? void 0 : ref12.call(_obj);
+(ref12 = o5()) === null || ref12 === void 0 ? void 0 : ref12();
```

Comment:
Is loose mode applied to `tsc` test suite?

Reply:
No, this is for fixing #2734, and sadly get intangled into this pr

Reply:
Then it's fine.
Thank you!